source/lucidity/template.py

```diff
@@ -252,7 +258,7 @@ def _construct_regular_expression(self, pattern):
             else:
                 _, value, traceback = sys.exc_info()
                 message = 'Invalid pattern: {0}'.format(value)
-                raise ValueError, message, traceback  #@IgnorePep8
+                raise ValueError(message, traceback)
```

Comment:
I think this would need a Python 2 / 3 switch.

Note: The current syntax is required as a workaround for exception chaining (to maintain original traceback information). I believe Python 3 supports this as `raise from`. 

https://docs.python.org/3/reference/simple_stmts.html#raise